firestore-shorten-urls-bitly/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## Version 0.1.3
+
+fixed - Fixed 406 HTTP error code from Bitly API due to `Content-Type` metadata not being set (#202).
```

Comment:
Pls PTAL this update to content - specifically `Content-Type` metadata part.

Reply:
It's not a metadata, it's an HTTP header. See https://www.geeksforgeeks.org/http-headers-content-type/ for explanation. (But no need to link to this, it's just for you to have context)

Reply:
thx!  fixed :-)